src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,42 @@
+import { Person } from '../types';
+import { PersonInfo } from './PersonInfo';
+
+interface Props {
+  people: Person[];
+}
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const getPersonByName = (name: string | null): Person | null => (name
```

Comment:
To improve performance, the `getPersonByName` function can be brought out as a separate helper outside of the component.